### PR TITLE
FIXES/COMPLETES ALL (new) DISEASES, /tg/ style cures, new level 0

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
-changes all of mark's diseases to work as intended
-adds a new symptom i planned to make a bounty of, which i figured out how to code myself. Spawns cockroaches and looks far, far more frightening than it is- good to get viro lynched if he's your target
-merges /tg/ style cures, with some cures changed due to cobbychems. may change more later, but this will do for now. No more gold and silver for cures
-rebalances viro traitor items
## Why It's Good For The Game
 makes the symptoms actually work as intended, no more useless symptoms
makes disease cures make sense, and harder to metagame
## Changelog
:cl:
add: New symptom: SBG syndrome. Looks spooky but is actually harmless
add: ported /tg/ style cures
tweak: changed inneffective symptoms to work as intended
balance: removed redundant viro traitor items and increased experimental disease cost
fix: fixed ducatopod and bee infestation's thresholds
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
alright, fucking finally, got travis to work and shit

i oughtta note- the virus cure from /tg/ is NOT stormcaster's unbalanced meme (https://github.com/BeeStation/BeeStation-Hornet/pull/420)

File: code/datums/diseases/advance/advance.dm [hunk 1]
```diff
@@ -34,10 +34,39 @@
 	var/oldres
 	// The order goes from easy to cure to hard to cure.
 	var/static/list/advance_cures = 	list(
-																	/datum/reagent/consumable/sodiumchloride, /datum/reagent/consumable/sugar, /datum/reagent/consumable/orangejuice,
-									/datum/reagent/medicine/spaceacillin, /datum/reagent/medicine/salglu_solution, /datum/reagent/consumable/ethanol,
-									/datum/reagent/medicine/leporazine, /datum/reagent/medicine/synaptizine, /datum/reagent/toxin/lipolicide,
-									/datum/reagent/silver, /datum/reagent/gold
+									list(	// level 1
```
Comment (qwertyquerty) [inline code/datums/diseases/advance/advance.dm]: These really don't need the multiline spacing

Comment (zeskorion) [reply code/datums/diseases/advance/advance.dm]: ok

Comment (zeskorion) [reply code/datums/diseases/advance/advance.dm]: Fails travis without multiline spacing. Considering this particular bit is a /tg/ port verbatim, I think it probably actually *does* need the multiline, else they'dve likely changed it.

Comment (zeskorion) [reply code/datums/diseases/advance/advance.dm]: Also: when this was ported by Stormcaster (#420), he made the port actually have even more multilining, and that was merged (even though it sucked)

**Comment by zeskorion** [comment]
*sigh*

This pr has been waiting awhile, just gonna make the curelist better without the fancy shit because qwert aint responding and the fixes here are important

File: code/datums/diseases/advance/symptoms/cockroach.dm [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+				M.visible_message("<span class='danger'>[M] squirms as a cockroach crawls from their pores!</span>", \
+								  "<span class='userdanger'>A cockroach crawls out of your face!!</span>")
+					new /mob/living/simple_animal/roach(M.loc)
+			if(prob(50))
+				to_chat(M, "<span class='notice'>You feel something crawling in your pipes!</span>")
+			if(death_roaches)
+  				on_death(A.affected_mob)
+					new /mob/living/simple_animal/roach(M.loc)
+						(M.loc)for(var/i in 1 to rand(1,5))
+       						new /mob/living/simple_animal/roach(M.loc)
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/cockroach.dm]: 🤔

**Comment by ike709** [comment]
Conflicts.

**Comment by zeskorion** [comment]
> Conflicts.

conflict is due to zanos' PR as it uses an unfixed version which has an incomplete version of the thing. ill change the internal name tho, see if that fixes it

**Comment by zeskorion** [comment]
nope, doesn't fix it. Zanos' pr isn't working as intended and actually causes the disease to knock you into a sugar coma. why you merged that one and not the one that has been sitting here for ten days is beyond me

**Review by eLeCtrOssSnake — COMMENTED** [review]
Definitely has some kind of logic uhhh, yes. 
Kind of a strange one

File: code/datums/diseases/advance/symptoms/cockroach.dm (hunk 2 above)
Comment (zeskorion) [reply code/datums/diseases/advance/symptoms/cockroach.dm]: What is the issue here?

**Comment by zeskorion** [comment]
thanks ike (This is not being sarcastic, by the way.)

**Comment by ike709** [comment]
YOLO.

**Pull request merged**